src/components/Login/EnterPhonePanel.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react'
+
+import Form from '@/components/Login/LoginForm'
+import Button from '@/components/UI/button'
+import { SelectOption } from '@/types/types'
+
+export interface EnterPhonePanelInterface {
+  onClick: (phoneNumber: string) => void
+}
+
+const EnterPhonePanel = ({ onClick }: EnterPhonePanelInterface) => {
+  const [displaynumber, setdisplaynumber] = useState('')
+  const [countrycode, setcountrycode] = useState('')
+
+  const countryCodeSelectOptions: SelectOption[] = [
+    { value: '+61', label: 'Australia (+61)' }
+  ]
+
+  function handleSubmit(event: React.ChangeEvent<HTMLFormElement>) {
+    onClick(countrycode + displaynumber)
+    event.preventDefault()
+    event.target.reset()
+  }
+
+  function formFilled() {
+    return displaynumber !== ''
```

Comment:
Phone number validation is difficult, please use a library like [libphonenumber-js](https://www.npmjs.com/package/libphonenumber-js) to confirm we get a valid number.